internal/transport/http2_server.go

```diff
@@ -456,6 +457,10 @@ func (t *http2Server) operateHeaders(frame *http2.MetaHeadersFrame, handle func(
 		return false
 	}
 
+	if encodings := mdata["grpc-accept-encoding"]; len(encodings) != 0 {
+		s.clientAdvertisedCompressors = strings.Join(encodings, ",")
+	}
+
```

Comment:
Is there a reason this cannot be moved to a `case` inside the `switch` statement in line 398 above?

Reply:
Done, moved it under the switch case